src/obfl-specification.html

```diff
@@ -1574,8 +1582,19 @@ <h4 id="L849" class="include">span</h4>
 
 <h4 id="L8491" class="include">style</h4>
 
-<p>The style element represents a segment of text with a specific style, e.g.
-emphasis or strong.</p>
+<p>The style element represents a segment of text with specific properties. An
+example of a text property is emphasis. An important precondition for a text
+property to be a style is that it could never apply to the whole document, only
+to a portion of it.</p>
+
+<!-- Another precondition for a property to be a style is that it is not a
```

Comment:
It is a bit strange to use a comment here. Please either put it in the specification, or leave it out.

Reply:
Hmm, I thought it was something worth a comment. It helps a bit to clarify things (for us). The reason I did not put it in the specification is obvious from the comment itself. There is a subtlety and it's not a good idea to put that in a specification. But it's so important to keep the comment. If both of you rather have that I remove it, it's OK.